kalite/topic_tools/__init__.py

```diff
@@ -259,6 +250,19 @@ def get_content_cache(force=False, annotate=False, language=settings.LANGUAGE_CO
         # Loop through all content items and put thumbnail urls, content urls,
         # and subtitle urls on the content dictionary, and list all languages
         # that the content is available in.
+        contents_folder = os.listdir(settings.CONTENT_ROOT)
+
+        subtitle_langs = {}
+
+        if os.path.exists(i18n.get_srt_path()):
+            for (dirpath, dirnames, filenames) in os.walk(i18n.get_srt_path()):
+                lc = dirpath.split("/")[-2]
```

Comment:
Just a guess, but the path returned by `i18n.get_srt_path()` must not have any subdirectories, right? If it did, then `lc = dirpath.split("/")[-2]` might not necessarily have the intended value when `os.walk` steps into those subdirectories. May be a non-issue.